README.md

```diff
@@ -18,8 +20,245 @@ some_object = SomeClass.new
 some_object.name = 'Some Name'
 some_object.amount = 11
 some_object.active = true
+
+puts some_object.inspect
+# => #<SomeClass:0x00555710f467c8 @name="Some Name", @amount=11, @active=true>
+
+puts some_object.to_h
+# => {:name=>"Some Name", :amount=>11, :active=>true}
+
+some_object.amount = 'foo' # => raises Schema::Attribute::TypeError
+```
+
+A special class is provided for `TrueClass` and `FalseClass` which is called
+`Boolean` to identify boolean values.
```

Comment:
I wouldn't call this a special "class," as it is not actually a class. Perhaps this:

"A special type called `Boolean` is provided which only allows the values `true` and `false`."

Reply:
Sounds good, right, it's just a constant